src/components/search/SearchResults.tsx

```diff
@@ -100,28 +93,72 @@ const InnerSearchResultList = <T extends SearchResultsType>(
     return router.query[param]
   }
 
+  const getSearchHits = createFetchDataFn<GetSearchResults_searchQuery_hits[]>([])({
```

Comment:
I think you can just not use the createFetchDataFn, because I created it to easily change between different data source.
In this case, I think it doesn't need to change between data source, just enable it when the graphql url is set and disable it when its not.

This is also related to the config, so you can make `enableSearch` false if the `GRAPHQL_URL` is not there
edit: I see you have implemented it 👍

Reply:
Implemented in [d88b004](https://github.com/dappforce/polkaverse/pull/87/commits/d88b0047f96955438d5d0f0204108d1617d72b8c)